src/components/deprecated-table/__tests__/molecules.test.tsx

```diff
@@ -1,10 +1,10 @@
 import { render } from '@testing-library/react'
 import {
-  TableHeadersRow,
-  TableHeader,
-  TableRow,
-  TableRowContainer,
-  TableCell,
+  DeprecatedTableHeadersRow,
+  DeprecatedTableHeader,
+  DeprecatedTableRow,
+  DeprecatedTableRowContainer,
+  DeprecatedTableCell,
   TableExpandableRowTriggerCell,
   TableExpandableRow,
   TableCtaTriggerCell,
```

Comment:
**question:** Not sure, but did we need to rename these as well?

Reply:
I do not find this useful for the current DS requirement, so I have only renamed the exports I may have intended to use.